scripts/explorer-mvs-configure.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+################################################################################
+# This program and the accompanying materials are made available under the terms of the
+# Eclipse Public License v2.0 which accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright IBM Corporation 2019
+################################################################################
+
+################################################################################
+# Variables required on shell:
+# - STATIC_DEF_CONFIG_DIR
+# - ROOT_DIR
+# - NODE_HOME
+
+. $ROOT_DIR/scripts/utils/validateNode.sh
```

Comment:
This should be in component validate.sh, or the run-zowe.sh, not in component configure.sh